VT-3 Preset Expansion/README.md

```diff
@@ -0,0 +1,41 @@
+# VT-3 Preset Expansion
+
+## Purpose
+
+The Purpose of this project is to expand the amount of voice presets for the Roland VT-3 Voice Transformer.  This is accomplished by interfacing the Novation Launchpad Mini with the the VT-3 through midi messages.
+
+-----
+
+## Installation
+
+1. Download the files
+2. Install the VT-3 Driver
+3. Plug in Launchpad Mini
+4. Plug in VT-3
+5. Run npm install in terminal
+6. Run npm start in terminal
+
+-----
+
+## Use
+
+The top row of circular buttons (1 - 8) represent "pages" on the program
+
+Pages 1 and 2 represent voice preset slots
+
+Page 3, when pressed, jumps to a random voice preset
+
+Pages 4 to 6 are empty
+
+Page 7 changes the selected preset's color
+
+Page 8 loads the current settings on the VT-3 into a preset you choose on the Launchpad
+
+-----
+
+## Resources
+
+[Launchpad Mini](https://global.novationmusic.com/launch/launchpad-mini#)
```

Comment:
Including documentation is awesome. Good work!

Reply:
Thanks!